tools/Dockerfile-tools

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:18.04
```

Comment:
Noting `ubuntu` base image, which is certainly familiar, but other images have moved to UBI-8.  Is there any value is using same here?

Reply:
@javabrett the reason this PR currently uses `ubuntu` is because the original `training-tools` Docker image had the same, and I wanted to ensure **exactly** the same behavior with packages like `date`.  I don't think there is a strong reason to have consistency with the CP images, and I don't have time to explore other options, so I prefer to leave this PR as-is with `ubuntu` unless there is a downside I'm not aware of.